src/searchReplace.tsx

```diff
@@ -286,13 +287,15 @@ function createTreeView(
   path: string,
   _commands: CommandRegistry,
   expandStatus: boolean[],
-  setExpandStatus: (v: boolean[]) => void
+  setExpandStatus: (v: boolean[]) => void,
+  onReplace: (r: IResults[]) => void
 ): JSX.Element {
   results.sort((a, b) => (a.path > b.path ? 1 : -1));
   const items = results.map((file, index) => {
     return (
       <TreeItem
         className="search-tree-files"
+        id={`tree-item-file-${index}`}
```

Comment:
Don't add an id because it will be inconsistent between search as index will change

```suggestion
```